packages/core/assemblyManager/assemblyManager.ts

```diff
@@ -28,12 +28,23 @@ function assemblyManagerFactory(conf: IAnyType, pm: PluginManager) {
        */
       assemblies: types.array(assemblyFactory(conf, pm)),
     })
+    .views(self => ({
+      get assemblyNameMap() {
+        const obj = {} as { [key: string]: Assembly }
+        for (const assembly of self.assemblies) {
+          for (const name of assembly.allAliases) {
+            obj[name] = assembly
+          }
+        }
+        return obj
+      },
+    }))
     .views(self => ({
       /**
        * #method
        */
       get(asmName: string) {
-        return self.assemblies.find(asm => asm.hasName(asmName))
+        return self.assemblyNameMap[asmName]
```

Comment:
this refactor to assemblyManager.get speeds up potentially calling assemblyManager.get here https://github.com/GMOD/jbrowse-components/pull/3725/files#diff-57f9ccf43c6a6dfb5af36c32f44b8bba8c6339a9e1a76409b44c26f2fefe5cebR71